docs/virtual_machines/disks_and_volumes.md

```diff
@@ -677,6 +677,8 @@ Kubernetes and provides two usage types:
 
 -   `Disk` a disk image must exist at a given location
 
+Note: you need to enable the HostDisk feature gate.
```

Comment:
@davidvossel is it `featuregate` or `feature gate`?  I want to start here with being consistent.

Reply:
I just reviewed [kubernetes](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)

We will follow suit and continue to use `feature gate` when referred to in a sentence.

I'll resolve this conversation and approve the pull req.